Bottom: 7b420678c5c74c1a1ab928455ab7373661dda7b4
Top:    dcb1bed3b69c333e28f3c419b2c9fa0078b1c532
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 02:13:58 +0100

Refresh of use-heap-structure-to-find-hol

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c719a30..5a0eabd 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1194,10 +1194,9 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
     }
   while ((curc = curc->f) != dest);
 
-  all_pa_info = malloc (sizeof (struct polyarea_info) * num_polyareas);
-
   /* make a polyarea info table */
   /* make an rtree of polyarea info table */
+  all_pa_info = malloc (sizeof (struct polyarea_info) * num_polyareas);
   tree = r_create_tree (NULL, 0, 0);
   i = 0;
   curc = dest;
